test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-5.xml

```diff
@@ -39,6 +39,7 @@
 	<variable name="currentMode" value="$mode204$"/>
 	<variable name="XXShareClassesEnableBCI" value="-XX:ShareClassesEnableBCI"/>
 	
+	<variable name="UTILSJAR" value="$UTILSDIR$$PATHSEP$utils.jar" />
 	<variable name="CP_HANOI" value="-cp $UTILSJAR$" />
  	<variable name="BOOTCP_HANOI" value="-Xbootclasspath/a:$UTILSJAR$" />
```

Comment:
It would appear `UTILSJAR` was already defined (see the use on line 44).

Reply:
It was defined in the playlist.xml but I replaced that definition with `UTILSDIR` since this test suite uses more than one file in that folder now.

Reply:
I missed that the variable was renamed. Thanks.